plugins/woocommerce/tests/php/src/Internal/CLI/Migrator/Mocks/MockPlatformFetcher.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Mock Platform Fetcher class for testing.
+ *
+ * @package Automattic\WooCommerce\Tests\Internal\CLI\Migrator\Mocks
+ */
+
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\Tests\Internal\CLI\Migrator\Mocks;
+
+use Automattic\WooCommerce\Internal\CLI\Migrator\Interfaces\PlatformFetcherInterface;
+
+/**
+ * A mock fetcher class for testing purposes.
+ */
+class MockPlatformFetcher implements PlatformFetcherInterface {
```

Comment:
Where are these used? Same for `MockPlatformMapper`? Should we clean up?

Reply:
@naman03malhotra  ⏫  ?